src/super_gradients/training/datasets/classification_datasets/image_net.py

```diff
@@ -0,0 +1,13 @@
+import torchvision.datasets as torch_datasets
```

Comment:
Please rename the file to imagenet_dataset.py.

Reply:
changed in 1cbca1572e70847368012225c96711905ac2fd9f